Add some additional help on google assistant trouble shooting (#14156)
home-assistant/home-assistant.github.io: source/_integrations/google_assistant.markdown

  4. Generate a new API key.
  5. Again, create a new project in the [Actions on Google console](https://console.actions.google.com/). Described above. But at the step 'Build under the Actions SDK box' choose your newly created project. By this, they share the same `project_id`.

#### 403 errors on request sync

> The `request_sync` service may fail with a 403 if the if the HomeGraph API is not enabled. Go to [Google API Console](https://console.cloud.google.com/apis/api/homegraph.googleapis.com/overview) and verify that HomeGraph API is enabled for your project

Reviewer: ```suggestion
The `request_sync` service may fail with a 403 if the HomeGraph API is not enabled. Go to [Google API Console](https://console.cloud.google.com/apis/api/homegraph.googleapis.com/overview) and verify that HomeGraph API is enabled for your project.
```

#### 404 errors on report state

> If you receove 404 errors linked to report state in your log, home assisting is reporting state for entites that where never synced to google. Ask your google home to `Sync my devices` or run the service `google_assistant.request_sync`
> 

Reviewer: ```suggestion
If you receive 404 errors linked to reporting state in your log, Home Assistant is reporting state for entities that were never synced to google. Ask your Google Home to `Sync my devices` or run the service `google_assistant.request_sync`.
```

#### Error during linking: "Could not update the setting. Please check your connection"

> Your fullfillment url may be invalid or unreachable. Recheck the `Fulfillment URL` as specified in [Manual Setup](#manual-setup) and verify that it's publically reachable.

Reviewer: ```suggestion
Your fulfillment URL may be invalid or unreachable. Recheck the `Fulfillment URL` as specified in [Manual Setup](#manual-setup) and verify that it's publically reachable.
```
